app/assets/stylesheets/application.scss

```diff
@@ -1,6 +1,8 @@
 $govuk-image-url-function: "image-url";
 $govuk-font-url-function: "font-url";
 $govuk-global-styles: true;
+$success-green: #00823b;
```

Comment:
Should this be re-using a constant from the design system?

Reply:
@fofr doesn't like the green that comes from govuk-frontend for "success". 
This green is consistent with manage courses ui and search and compare ui 👌

Reply:
Actually the issue here seems to be that the govuk green used for buttons is [inside the scope of the SASS button component](https://github.com/alphagov/govuk-frontend/blob/c62ea3a15d15bfac42461cfc77365dbe342de651/package/components/button/_button.scss#L7) and as such is not directly importable. @fofr mentioned there is a discussion around this.

It's pragmatic to duplicate this in our own code for now. :)